### ISSUE TITLE
Images don't show in in empty chapters

### ISSUE THREAD

**Issue by dumbledore** [dumbledore/AlbiteREADER#168]

**Comment by dumbledore** [comment]
If the image doesn't have alt text, it won't be displayed because of the current implementation of the `isEmpty()` method didn't take image into account.
